srcs/pressure_controller.cpp

```diff
@@ -455,7 +463,7 @@ void PressureController::updatePhase(uint16_t p_tick) {
 #if VALVE_TYPE == VT_FAULHABER
         /* -5 mmH2O is added to prevent plateau not detected in case the pressure is almost reached.
          This is mandatory to help the blower regulation to converge */
```

Comment:
This should probably be updated(?)

Reply:
Yes it should be : /* -5 mmH2O is added to prevent peak pressure not reached in case the pressure is almost reached. This is mandatory to help the blower regulation to converge */

Reply:
OK! I'm fixing it and merging this PR 🙂